src/main/java/au/org/ala/biocache/service/AssertionService.java

```diff
@@ -180,4 +181,84 @@ private UserAssertions getCombinedAssertions(UserAssertions existingAssertions,
 
         return combined;
     }
+
+    // * There are 5 states for User Assertion Status:
+    // * NONE: If no user assertion record exist
+    // * UNCONFIRMED: If there is a user assertion but has not been verified by Collection Admin
+    // * OPEN ISSUE: Collection Admin verifies the record and flags the user assertion as Open issue
+    // * VERIFIED: Collection Admin verifies the record and flags the user assertion as Verified
+    // * CORRECTED: Collection Admin verifies the record and flags the user assertion as Corrected
+    private void UpdateUserAssertions(String recordUuid, UserAssertions userAssertions) throws IOException {
```

Comment:
im not sure why `UpdateUserAssertions` is starting with an upper case letter. Coding convention should be lowercase.